doc/case-studies/timelock/env.tla

```diff
@@ -0,0 +1,83 @@
+(*
+ * Model of Soroban environment
+ * 
+ * Andrey Kuprianov, 2024
+ *)
+
+---- MODULE env ----
+
+EXTENDS Integers, Sequences, Apalache
+
+(*
+    @typeAlias: address = Str;
+    @typeAlias: dataKey = Str;
+*)
+typedefs == TRUE
+
+VARIABLES
```

Comment:
There's a broader issue here with how this is defined: If `env`, `tx` and `instance_strorage` are to be defined as TLA state variables, then instrumentation would have to instantiate them all, which is not what happens (see [here](https://github.com/freespek/solarkraft/blob/81cf373ef8d1523e4458e49bf447cfcf10e726f6/solarkraft/src/instrument.ts#L23-L69))
The `env` gets passed as the 1st function argument, the data in `tx` is not instrumented at the moment, and neither is `instance_storage`, though the latter would probably be the easiest to simply add in.